src/components/app/list-new/AppList.tsx

```diff
@@ -1015,6 +1017,7 @@ export default function AppList({ isSuperAdmin, appListCount, isArgoInstalled }:
                             setAppCount={setAppCount}
                             updateDataSyncing={updateDataSyncing}
                             isArgoInstalled={isArgoInstalled}
+                            setCurrentAppName={setCurrentAppName}
```

Comment:
Can we not consume the context directly in the component? Check the GlobalConfigurationProvider for usage with class components

Reply:
We are consuming it with custom hook 🪝  so wanted to keep it same